CMakeLists.txt

```diff
@@ -81,6 +81,8 @@ endif(REQUIRED_BOOST_LIBRARIES)
 
 add_definitions("-DJSONV_TEST_DATA_DIR=\"${CMAKE_SOURCE_DIR}/src/jsonv-tests/data\"")
 
+configure_file(libjsonv.pc.in libjsonv.pc)
```

Comment:
Can you also add this file to `.gitignore` so nobody accidentally commits the generated one?